.gitignore

```
@@ -1,3 +1,2 @@
-node_modules
 coverage
-.idea
\ No newline at end of file
+.idea
```

Why have you removed the node_modules? As it has a package.json file it also will have a node_modules file as soon as `npm install` is called.
```suggestion
node_modules
.idea
```

---

readded